docs/tf/ray_usage.rst

```diff
@@ -250,11 +250,11 @@ Ray Usage Example for Node Classification with GAT
     ...     train_loop_config={
     ...         "batch_size": 2708,
     ...         "data_dir": data_dir.name,
-    ...         "n_epochs": 100,
+    ...         "n_epochs": 300,
```

Comment:
why do we need this? I thought training should be completely deterministic.

Reply:
Not in tf/keras, it is not possible to properly seed it but higher epochs helps with stability.

Reply:
https://keras.io/getting_started/faq/#how-can-i-obtain-reproducible-results-using-keras-during-development

Reply:
Ah even better thank you

Reply:
This improves stability but still 1/3 is == .5.